libraries/api/src/index.js

```diff
@@ -98,6 +96,12 @@ class APIBuilder {
    *   ].join('\n'),
    *   cleanPayload: payload => payload,           // function to 'clean' the payload for
    *                                               // error messages (e.g., remove secrets)
+   *   implicitServiceScopes: false,               // defaults to true
+   *                                               // when true, the single scope
+   *                                               // 'service:<serviceName>:<name>'
```

Comment:
I would like to avoid this scope pattern.  It differs from our existing `<service>:<action>:<args>` pattern (and uses camelCase instead of dash-case), leaving us either to explain why we have two half-implemented scope patterns, or migrate to one or the other.  Instead, let's define a scope for each otherwise-anonymous API method that matches other scopes for that service.  There aren't that many, and I'm sure in almost every case the appropriate scope is pretty easy to discern.  Where it's not easy to discern, `<service>:<dash-cased-action>` is a good fallback.

Reply:
I agree, good call.